helm/cluster-apps-operator/templates/configmap.yaml

```diff
@@ -30,6 +30,10 @@ data:
           keyFile: ''
       provider:
         kind: {{ .Values.provider.kind }}
+      proxy:
+        noProxy: {{ .Values.proxy.noProxy }}
+        http: {{ .Values.proxy.http }}
+        https: {{ .Values.proxy.https }}
```

Comment:
is there a reason why we change the naming here? I would rather be consistent and use `no_proxy`, `http_proxy` and `https_proxy` as everywhere else

Reply:
i've tooked [`chart-operator`](https://github.com/giantswarm/chart-operator/pull/961/files) and `kyverno-policies-connectivity` as reference where the values are called that way.